config/initializers/session_store.rb

```diff
@@ -0,0 +1,3 @@
+# Be sure to restart your server when you modify this file.
+
+Rails.application.config.session_store :cookie_store, key: '_sideci_rails_sandbox_session'
```

Comment:
__[rubocop]__
* Line is too long. [90/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->